### DISCUSSION TITLE
What do you know about Rest API

### DISCUSSION THREAD

**Discussion by omphilejmatsobe** [community/community#56871] [Apps, API and Webhooks] [answered]
### Select Topic Area

Product Feedback

### Body

I have heard some developers talk about how they use Rest API and I keep getting lost. What is Rest API and is it a must for every developer to use it?

**Comment by mrepol742** [comment]
Rest API is an application programming interface (API) that uses HTTP requests to GET, PUT, POST and DELETE data. It's a must for every developer to use it because it makes it easier to access and manipulate data.

**Comment by RazvanCretu** [comment, marked as answer]
👋🏽 

As @mrepol742 stated, API stands for Application Programming Interface and REST stands for REpresentational State Transfer.

Basically it is a must for every developer as all apps use APIs to acces data stored on remote servers or dbs.

Take me posting this comment as an example. 

I click the comment button, the browser executes some javascript code so that it takes my input and sends a request to another server(another computer basically) to process and store my input in a database so other users could see it at a later time.

Hope it helps! 👐

**Reply by omphilejmatsobe** [reply]
Thank you for the explanation 👍👍
